templates/CMakeLists.txt.template

```diff
@@ -40,17 +40,17 @@
   # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
   <%!
-  
+
```

Comment:
The `gRPC_PROTOBUF_PROVIDER=module` now mean compile and use `protobuf` from `git` source code submodule.
I think the `gRPC_PROTOBUF_PACKAGE_TYPE` with values `MODULE`, `CONFIG` or empty `` or `AUTO` to using both will be useful.